src/applications/personalization/profile/tests/unit-test-helpers.js

```diff
@@ -29,6 +32,16 @@ export function renderWithProfileReducers(
   });
 }
 
+export function renderWithProfileReducersAndRouter(
+  ui,
+  { initialState = {}, reducers = {} } = {},
+) {
+  return renderWithStoreAndRouter(ui, {
+    reducers: { ...profile, connectedApps, ...reducers },
+    initialState,
+  });
+}
+
```

Comment:
If you need to render a profile component that relies on react router, redux store, and the specific profile reducers, then you can use this and it will help for unit tests to prevent rendering errors.